docs/content/rules/sort-imports.mdx

```diff
@@ -201,6 +201,12 @@ Specifies a maximum line length for sorting imports. When the line length exceed
 
 This option is only available when the type is set to `'line-length'`.
 
+### tsConfigRootDir
+
+<sub>default: `undefined`</sub>
```

Comment:
@hugop95, what do you think about defaulting to something else, like e.g.:
- `languageOptions.parserOptions.tsconfigRootDir` (<=> `undefined` if not defined)
- `import.meta.dirname` (with the possibility to override with `null` if someone isn't interested in such an import sorting)

see https://typescript-eslint.io/getting-started/typed-linting/

Reply:
@OlivierZal The option I set is a bit of a placeholder at the moment, but those are great suggestions.

Reply:
On the other hand, I don't think we should activate the feature by default (and try importing typescript + searching for a `tsconfig.json` file). So I'm not fully sure about option 2. 🧐

Reply:
EDIT: we probably should disable by default to avoid any performance issue.